src/ontology/cl-edit.owl

```diff
@@ -31685,6 +31688,50 @@ AnnotationAssertion(rdfs:label obo:CL_4033056 "onychocyte")
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19648928") Annotation(oboInOwl:hasDbXref "PMID:26390871") Annotation(oboInOwl:hasDbXref "PMID:34156490") Annotation(oboInOwl:hasDbXref "https://nrs.harvard.edu/URN-3:HUL.INSTREPOS:37368328") Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0001-6677-8489") obo:IAO_0000115 obo:CL_4033058 "A luminal epithelial cell of the mammary gland that transduces endocrine cues to orchestrate proliferation, architectural remodeling, and differentiation of other cells in the mammary gland via paracrine signaling. In humans, a luminal hormone-sensing cell can be identified by high levels of EPCAM and low levels of CD49f, and in mice, it can be identified by low levels of CD29 and high levels of Foxa1, CD133, and Sca1 (Ly6a).")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033058 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033058 <https://orcid.org/0000-0003-4969-670X>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033058 "2023-10-16T08:30:25Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://nrs.harvard.edu/URN-3:HUL.INSTREPOS:37368328") oboInOwl:hasBroadSynonym obo:CL_4033058 "ER-positive luminal hormone-sensor cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1101/2023.04.21.537845") oboInOwl:hasBroadSynonym obo:CL_4033058 "hormone sensing cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26390871") oboInOwl:hasBroadSynonym obo:CL_4033058 "hormone-sensing mammary epithelial cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37380767") oboInOwl:hasBroadSynonym obo:CL_4033058 "luminal hormone-responsive cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20346151") oboInOwl:hasBroadSynonym obo:CL_4033058 "luminal mature cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1101/2023.04.21.537845") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033058 "HS")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26390871") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033058 "HS-MEC")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0001-6677-8489") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033058 "LHS")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37380767") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033058 "LumHR")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26390871") Annotation(oboInOwl:hasDbXref "PMID:34156490") Annotation(oboInOwl:hasDbXref "https://orcid.org/0000-0003-4969-670X") rdfs:comment obo:CL_4033058 "Following lineage specification, a luminal hormone-sensing cell expresses receptors for one or more hormones, such as estrogens, progesterone, androgens, and prolactin.
```

Comment:
Typically comments do not include multiple paragraphs. Unsure if this will cause downstream formatting issues.

Reply:
Merged all in one paragraph